<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gerenciador do inventário de peças</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>761</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 30pt &quot;Century Gothic&quot;;
color: rgb(255, 85, 0);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Peças de carro</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>771</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_12">
     <attribute name="title">
      <string>Detalhes do inventário</string>
     </attribute>
     <widget class="QTableWidget" name="tabela">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>701</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Referência</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Peça</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Quantidade</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Preço(R$)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Marca</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>541</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Pesquisa de referência com quantidade em estoque menor ou igaul a:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="numero_botao">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>10</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Century Gothic&quot;;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="procura_botao">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Procurar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_11">
     <attribute name="title">
      <string>Estatísca do Inventário</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Número de referências</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Número de peças diferentes</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>30</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Peça de menor valor</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>100</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Peça de maior valor</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_ref">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;
color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_peca">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>100</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;
color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_menor">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>30</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;
color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_maior">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>100</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;
color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_ref_menor">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>30</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;
color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string>REF</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_ref_maior">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>100</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;
color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string>REF</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tabela2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>561</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <column>
       <property name="text">
        <string>Referência</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Peça</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Quantidade</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>471</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>TOP 3 peças com menor quantidade em estoque</string>
      </property>
     </widget>
     <widget class="QPushButton" name="verificar_inventario">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Verificar inventário</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_10">
     <attribute name="title">
      <string>Editar inventário</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Referência</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Peça</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Quantidade</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>60</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Preço(R$)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>100</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Marca</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="peca">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;
color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="ref">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;
color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="marca">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>100</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;
color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="preco">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>60</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;
color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="id">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;
background-color: rgb(170, 165, 159);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QSpinBox" name="quantidade">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;
color: rgb(255, 85, 0);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="next_btn">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>300</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="last_btn">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>300</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string>&gt;&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="first_btn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string>&lt;&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="previous_btn">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>300</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="update_btn">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>300</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;
color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_btn">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>300</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;
color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string>Deletar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_btn">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>300</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;
color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string>Adicionar</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="atualizar">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>500</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Atualizar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>752</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
